src/integrate/concat/config.vsh.yaml

```diff
@@ -33,9 +33,9 @@ functionality:
       type: string
       description: The compression format to be used on the final h5mu object.
       default: "gzip"
-    - name: "other_axis_mode"
+    - name: "--other_axis_mode"
       type: string
-      choices: [same, unique, first, only, concat]
+      choices: [same, unique, first, only, concat, move]
       default: concat
```

Comment:
Should the default be `move`? I vote for yes :)

Reply:
Done!